src/inc/clrnt.h

```diff
@@ -789,6 +789,8 @@ typedef struct _DYNAMIC_FUNCTION_TABLE {
 //
 #ifdef _TARGET_AMD64_
 
+#define COMPACT_RUNTIME_FUNCTION        0
```

Comment:
Ah, that makes sense then. So can you please just modify the COMPACT_RUNTIME_FUNCTION to something like RUNTIME_FUNCTION_HAS_ENDADDRESS or RUNTIME_FUNCTION_NEEDS_ENDADDRESS or something like that?